rispy/parser.py

```diff
@@ -295,13 +299,14 @@ def load(
 
     Args:
         file (Union[TextIO, Path]): File handle to read ris formatted data.
+        encoding(str, optional): File encoding, only used when a Path is supplied.
```

Comment:
add "Consistent with the python standard library, if None is supplied, the default system encoding is used"